.circleci/config.yml

```diff
@@ -262,6 +264,20 @@ jobs:
 
       - store_artifacts:
           path: conda_env_dump.log  # This is the list of modules in conda and pip
+
+      - run:
+          name: Run HITL functionality
+          no_output_timeout: 60m
```

Comment:
It looks like your recent successful run took about 4 minutes. I'm actually shocked it downloaded HSSD so quickly! Anyway let's reduce this timeout to something like 10 or 15 minutes.